### PR TITLE
fix: some of the types are updated

### PR DESCRIPTION
**Pull request by palashgdev**
updated some of the types in the query builder

### CODE REVIEW

**Review by yeshev — CHANGES_REQUESTED** [review]

**Review by techchintan — APPROVED** [review]

File: frontend/src/container/QueryBuilder/filters/GroupByFilter/GroupByFilter.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ export const GroupByFilter = memo(function GroupByFilter({
 
 	const { data, isFetching } = useQuery(
 		[QueryBuilderKeys.GET_AGGREGATE_KEYS, searchText],
-		async () =>
+		() =>
 			getAggregateKeys({
```
[the remark this answers is not in the document]
Comment (techchintan) [reply frontend/src/container/QueryBuilder/filters/GroupByFilter/GroupByFilter.tsx:29]: It is not required for now. as it has enabled Boolean to have different values. We have already created a service which is enough.
CC: @palashgdev @yeshev

[the remark this answers is not in the document]
Comment (palashgdev) [reply frontend/src/container/QueryBuilder/filters/GroupByFilter/GroupByFilter.tsx:29]: yeah that's the reason i didn't pointed out when feature was developing

File: frontend/src/container/QueryBuilder/filters/QueryBuilderSearch/index.tsx [hunk 2]
```diff
@@ -1,22 +1,21 @@
-import { Select, Spin, Tag, Tooltip } from 'antd';
+import { Select, Spin } from 'antd';
 import { useAutoComplete } from 'hooks/queryBuilder/useAutoComplete';
-import React, { useEffect, useMemo } from 'react';
+import React, { useCallback, useEffect, useMemo } from 'react';
```
Comment (GermaVinsmoke) [inline frontend/src/container/QueryBuilder/filters/QueryBuilderSearch/index.tsx:3]: From `v17` onwards, you can skip importing `React`

`import { useCallback, useEffect, useMemo } from 'react';` will also work

Comment (palashgdev) [reply frontend/src/container/QueryBuilder/filters/QueryBuilderSearch/index.tsx:3]: yeah we need to update eslint for this one

**Comment by palashgdev** [comment]
not needed now

**Pull request closed without merging**